spec/Accessport2LtpTranslator+forwardings.yaml

```diff
@@ -29,377 +48,385 @@ forwardings:
       - server-name: /v1/start-application-in-generic-representation
-        uuid: atlt-1-0-0-op-s-bs-000
+        uuid: atlt-1-0-2-op-s-bs-000
       - server-name: /v1/inform-about-application
-        uuid: atlt-1-0-0-op-s-bs-001
+        uuid: atlt-1-0-2-op-s-bs-001
       - server-name: /v1/inform-about-application-in-generic-representation
-        uuid: atlt-1-0-0-op-s-bs-002
+        uuid: atlt-1-0-2-op-s-bs-002
       - server-name: /v1/inform-about-release-history
-        uuid: atlt-1-0-0-op-s-bs-003
+        uuid: atlt-1-0-2-op-s-bs-003
       - server-name: /v1/inform-about-release-history-in-generic-representation
-        uuid: atlt-1-0-0-op-s-bs-004
+        uuid: atlt-1-0-2-op-s-bs-004
       - server-name: /docs
-        uuid: atlt-1-0-0-op-s-bs-005
+        uuid: atlt-1-0-2-op-s-bs-005
       - server-name: /v1/translate-equipment-sequence-ids-to-ltp-uuids
-        uuid: atlt-1-0-0-op-s-is-000
+        uuid: atlt-1-0-2-op-s-is-000
     consequent-requests:
       - client-name: EATL://v1/record-service-request
-        uuid: atlt-1-0-0-op-c-bs-eatl-2-0-1-000
+        uuid: atlt-1-0-2-op-c-bs-eatl-2-1-2-000
 
   - forwarding-name: OamRequestCausesLoggingRequest
-    uuid: atlt-1-0-0-op-fc-bm-003
+    uuid: atlt-1-0-2-op-fc-bm-003
     forwarding-type: InvariantProcessSnippet
     management-requests:
       operation-client-update:
         - server-name: /v1/redirect-oam-request-information
-          uuid: atlt-1-0-0-op-s-bm-003
+          uuid: atlt-1-0-2-op-s-bm-003
       fc-port-update:
         - server-name: /v1/redirect-oam-request-information
-          uuid: atlt-1-0-0-op-s-bm-003
+          uuid: atlt-1-0-2-op-s-bm-003
       fc-port-deletion:
       operation-client-deletion:
     initiating-requests:
     consequent-requests:
       - client-name: OamLog://v1/record-oam-request
-        uuid: atlt-1-0-0-op-c-bs-ol-2-0-1-000
+        uuid: atlt-1-0-2-op-c-bs-ol-2-1-2-000
 
   - forwarding-name: BasicAuthRequestCausesInquiryForAuthentication
-    uuid: atlt-1-0-0-op-fc-bm-005
+    uuid: atlt-1-0-2-op-fc-bm-005
     forwarding-type: InvariantProcessSnippet
     management-requests:
       operation-client-update:
         - server-name: /v1/inquire-basic-auth-approvals
-          uuid: atlt-1-0-0-op-s-bm-012
+          uuid: atlt-1-0-2-op-s-bm-012
         - server-name: /v1/inquire-oam-request-approvals
-          uuid: atlt-1-0-0-op-s-bm-005
+          uuid: atlt-1-0-2-op-s-bm-005
       fc-port-update:
         - server-name: /v1/inquire-basic-auth-approvals
-          uuid: atlt-1-0-0-op-s-bm-012
+          uuid: atlt-1-0-2-op-s-bm-012
         - server-name: /v1/inquire-oam-request-approvals
-          uuid: atlt-1-0-0-op-s-bm-005
+          uuid: atlt-1-0-2-op-s-bm-005
       fc-port-deletion:
       operation-client-deletion:
     initiating-requests:
       # [all oam requests]
     consequent-requests:
       - client-name: AdministratorAdministration://v1/approve-oam-request
```

Comment:
client-name shall be AccessAdministration:://v1/approve-basic-auth-request

Reply:
fixed